OracleRestDataServices/dockerfiles/Dockerfile

```diff
@@ -50,7 +50,11 @@ RUN mkdir -p  $ORDS_HOME/doc_root && \
     rm $INSTALL_FILE && \
     mkdir -p $ORDS_HOME/config/ords && \
     java -jar $ORDS_HOME/ords.war configdir $ORDS_HOME/config && \
-    chown -R oracle:dba $ORDS_HOME
+    chown -R oracle:dba $ORDS_HOME && \
+    yum install -y tar && \
+    yum install -y unzip && \
```

Comment:
we need tar for copying apex.zip to ORDS container using `kubectl cp` as we need APEX images to be present on ORDS container to front APEX using ORDS

Reply:
We need tar because `kubectl cp` requires tar
!!!Important Note!!! # Requires that the 'tar' binary is present in your container # image. If 'tar' is not present, 'kubectl cp' will fail
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands

Reply:
The `kubectl cp` command is just a synonym for a `kubectl exec` command that uses `tar`. You could construct a `kubectl exec` command using `jar` that would achieve the same effect of copying files in to the pod.

Reply:
There are examples of this conversion in the help for the `kubectl cp` command: https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/cp/cp.go